README.md

```diff
@@ -831,6 +833,9 @@ See **[Ticketing](#ticketing)**
   * [Microglark](http://micro.glark.io) - A hacky minimalistic remote pair programming editor. ([Demo](http://micro.glark.io), [Source Code](https://github.com/Bluefinch/microglark)) `AGPLv3` `Javascript`
   * [Regexr](http://regexr.com/) - RegExr is a HTML/JS based tool for creating, testing, and learning about Regular Expressions. ([Source Code](https://github.com/gskinner/regexr/)) `MIT` `Nodejs`
   * [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio/)) `AGPLv3` `Java/C++`
+  * [Visual Studio Community 2016](https://www.visualstudio.com/vs/community/) - A free, fully-featured and extensible Windows-based IDE for building applications that can target Windows, Android, and iOS,
```

Comment:
Judging from https://beta.visualstudio.com/vs/community/ it does not provide a web IDE or other network services, therefore this doesn't qualify for the list. License and Language are also missing.